### ISSUE TITLE
Status 400 all the time

### ISSUE THREAD

**Issue by ZIV-LY** [wrenger/obsidian-languagetool#1]
No matter the standard or the premium endpoint, the status is 400 all the time.

**Comment by ZIV-LY** [comment]
The original plugin obsidian-languagetool-plugin can work normally.

**Comment by kudlav** [comment]
After installation the default settings are set to custom URL "standard" and the check command does nothing.
![image](https://github.com/user-attachments/assets/b8d2f380-e981-444b-9299-588fefe14712)

Changing to Standard:
```
23/07/2024, 15:38:06:
Error: 'Request to LanguageTool failed: Please check your connection and server URL.
Error: Request failed, status 400'
Settings: {"serverUrl":"https://api.languagetool.org","autoCheckDelay":3000,"shouldAutoCheck":true,"languageVariety":{},"pickyMode":false,"username":"REDACTED","apikey":"REDACTED"}
```

Changing to Premium
```
23/07/2024, 15:39:04:
Error: 'Request to LanguageTool failed: Please check your connection and server URL.
Error: Request failed, status 413'
Settings: {"serverUrl":"https://api.languagetoolplus.com","autoCheckDelay":3000,"shouldAutoCheck":true,"languageVariety":{},"pickyMode":false,"username":"REDACTED","apikey":"REDACTED"}

```

**Comment by XIIMarcel** [comment]
I'm also having trouble connecting.

I have already stored the URL "api.languagetoolplus.com/v2" from the API page.
But as soon as the plugin tries to establish a connection after a few seconds to check the text, the error message appears.
API user name and key are stored.

However, the synonym function works.

Error message for the Default Premium URL:

`4.8.2024, 17:44:48:
Error: 'Request to LanguageTool failed: Please check your connection and server URL.
Error: Request failed, status 400'
Settings: {"serverUrl":"https://api.languagetoolplus.com","autoCheckDelay":3000,"shouldAutoCheck":true,"languageVariety":{},"pickyMode":false,"username":"REDACTED","apikey":"REDACTED","synonyms":"de"}
`

URL of the API page:

`4.8.2024, 16:38:01:
Error: 'Request to LanguageTool failed: Please check your connection and server URL.
Error: Request failed, status 404'
Settings: {"serverUrl":"https://api.languagetoolplus.com/v2","autoCheckDelay":3000,"shouldAutoCheck":true,"languageVariety":{},"pickyMode":false,"username":"REDACTED","apikey":"REDACTED","synonyms":"de"}
`

Obsidian Version: 1.6.7
Plugin: 1.0.5

**Comment by wrenger** [comment]
This might be caused by an invalid language configuration. I hope 61bc2c6 fixes this.

**Comment by wrenger** [comment]
I've released these fixes as 1.0.6. Does this solve your problem?

**Comment by XIIMarcel** [comment]
Now the plugin is working again. Thank you.
